android/capacitor.settings.gradle

```diff
@@ -2,5 +2,8 @@
 include ':capacitor-android'
 project(':capacitor-android').projectDir = new File('../node_modules/@capacitor/android/capacitor')
 
+include ':capacitor-toast'
+project(':capacitor-toast').projectDir = new File('../node_modules/@capacitor/toast/android')
+
 include ':proofme-id-sdk'
-project(':proofme-id-sdk').projectDir = new File('../node_modules/@proofme-id/sdk/./web/reader/android')
+project(':proofme-id-sdk').projectDir = new File('../../Libraries/Proofme-SDK/./web/reader/android')
```

Comment:
This will break the build!
You need to make sure that the sdk is installed by npm, and not linked to local filesystem.